README.md

```diff
@@ -162,6 +162,20 @@ The following work on normal and visual modes:
 
 -   `:Tocv`: Same as `:Toc` for symmetry with `:Toch` and `Tocv`.
 
+## Issue reporting
+
+Issues are tracked within GitHub.
+
+When reporting issues, your report is more effective if you include a minimal example file that reproduces the problem. Try to trim out as much as possible, until you have the smallest possible file that still reproduces the issue. Paste the example inline into your issue report, quoted using four spaces at the beginning of each line, like this example from issue [#189](https://github.com/plasticboy/vim-markdown/issues/189):
```

Comment:
I'd shorten this to:

Provide a minimal buffer test case if applicable. Try to trim out as much as possible, until you have the smallest possible file that still reproduces the issue.

Include the actual buffer text, not just an image, as it makes it easier for others to reproduce the problem.